src/main/java/gift/domain/BaseTimeEntity.java

```diff
@@ -0,0 +1,27 @@
+package gift.domain;
+
+import jakarta.persistence.EntityListeners;
+import jakarta.persistence.MappedSuperclass;
+import java.time.LocalDateTime;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+@MappedSuperclass
+@EntityListeners(AuditingEntityListener.class)
+public class BaseTimeEntity {
```

Comment:
오 생성일과 수정일 기록에 대한 클래스 추출 너무 좋습니다!
해당 클래스는 객체로 만들어질 수 없도록 abstract 선언하면 더욱 좋을 것 같습니다ㅎㅎ
```suggestion
abstract class BaseTimeEntity {
```

Reply:
넵 수정하겠습니다!